### PR TITLE
Fix delete dashboard

### PR DESCRIPTION
**Pull request by GowthamShanmugam**

### CODE REVIEW

File: tendrl/gluster_integration/message/callback.py [hunk 1]
```diff
@@ -501,7 +501,10 @@ def volume_delete(self, event):
                                              fqdn,
                                              path
                                          )
-
+                            # fetch brick_path to remove dashboard
+                            brick_full_path = NS._int.wclient.read(
+                                "%s/brick_path" % brick_path
```
Comment (r0h4n) [inline tendrl/gluster_integration/message/callback.py]: put this in a try/except , we dont want issues in one brick stop the entire volume delete

**Comment by r0h4n** [comment]
Verified?

File: tendrl/gluster_integration/message/callback.py (hunk 1 above)
Comment (GowthamShanmugam) [reply tendrl/gluster_integration/message/callback.py]: i will change it

Comment (GowthamShanmugam) [reply tendrl/gluster_integration/message/callback.py]: It won't fail, it will take next volume when it fails, This code is under try block

**Comment by GowthamShanmugam** [comment]
@r0h4n verified

**Pull request merged**